_sass/components/partners/_partners.scss

```diff
@@ -14,6 +14,7 @@
 }
 
 .partners-body {
+    max-width: 640px;
```

Comment:
maybe for another time, would we want to consider defining this based on character size? [USWDS does 78ex](https://designsystem.digital.gov/components/prose/) and there are some styles that use `rem` too

Reply:
When I replaced `partners-body` with `usa-prose`, it seemed a bit of overcorrection. Lopsided if there isn't another element next to it. 

However, `usa-prose` might work well if we are implementing the side nav.

|640px| usa-prose|
|---|---|
| ![localhost_4000_partners_rrb-impact-story_](https://user-images.githubusercontent.com/6327082/172734348-0837efe5-f2e3-4925-a5c8-94a9ba0f09d4.png) | ![localhost_4000_partners_rrb-impact-story_ (2)](https://user-images.githubusercontent.com/6327082/172734407-c3c3c64a-6c0b-4fcb-a900-1bc08875e9c2.png) |

Reply:
@aduth Would that affect other static sites, like the brochure site? We could consider `5` or `6` if it doesn't create a lopsided effect. 640px is a suggestion that has a balance between close to optimal line length while keeping the layout intact. So, open to ideas. //cc @cdelivuk if you have any preferences

Reply:
@nickttng It depends where we implement it, I think. If we implement it as the default setting in the design system, it would apply across all sites. If we implement it in this repository, it would apply to the brochure site and the partners site (since they're the same codebase).

That being said, I couldn't find a single instance of `usa-prose` in this repository, so it doesn't look like we're currently using it anyways? (i.e. no risk of breaking anything existing)

Reply:
After testing with `5` (`74ex`), I think it looks decent and similar to 640px. 

| 74ex | 88ex |
|---|---|
| ![usa-prose-74ex](https://user-images.githubusercontent.com/6327082/172923426-0bf0a807-109e-4184-8f8e-127ecb3e5a3f.png) | ![usa-prose-88ex](https://user-images.githubusercontent.com/6327082/172923470-cb7fe95c-af8e-419f-9ed8-430c26e69b41.png) |

@aduth @cdelivuk @zachmargolis What do you think about us merging for now with 640px. Then follow up in the future to test with `usa-prose` (adjusted to use `5`) since it'd require further review for both brochure and partner content?

Reply:
yeah, i wonder how these work with the side nav. i think we can go with merging for now then follow up!